### ISSUE TITLE
Override Hyperbole

### ISSUE THREAD

**Issue by NicolasMassart** [errata-ai/proselint#1]
Hi,
my markdown code contains some syntaxes like :
```
!!! important
    the important text
```
that is seen as hyperbolic syntax like "it's so cool!!!"
But it's not the case, it's a false positive as this is the syntax for Admonition plugin https://squidfunk.github.io/mkdocs-material/extensions/admonition/ that I would like Vale not to consider as an issue.
I tried overriding the Hyperbole file:
```
extends: existence
message: "'%s' is hyperbolic."
level: error
nonword: true
tokens:
  - '[a-z]*[!]{2,}'
  - '[a-z]*\?{2,}'
```
but I can't find a way to write the tokens regex to prevent matching my specific syntax. I tried things like `[a-z]+[!]{2,}` or `[\w ]+[!]{2,}` that works when I test them in other regexp testers but Vale always take my Admonition markers in account and raises errors.
Any idea?
Thanks.

**Comment by jdkato** [comment]
Thanks for the report -- I've fixed the issue in ea64c1ab6803b837584a600aaf0729cba359f635 and it is available in [`v0.3.1`](https://github.com/errata-ai/proselint/releases/tag/v0.3.1)

To perform an "override," you need to copy one of the releases from this repo into your `StylesPath` directory and then include `BasedOnStyles = proselint` in your config file.
